subgraphs/smolverse-chop-shop/schema.graphql

```diff
@@ -0,0 +1,47 @@
+"Collection of upgrades"
+type Upgrade @entity {
+  "Singleton with ID `only`"
+  id: ID!
```

Comment:
Take a look at one of the newer subgraphs like `bridgeworld-approvals` for reference, but there is a new field type we can use for IDs called `Bytes` and it speeds up syncing times.